docs/en/tutorials/how_to.md

```diff
@@ -0,0 +1,64 @@
+# Tutorial 11: How to xxx
+
+This tutorial collects answers to any `How to xxx with MMDetection`. Feel free to update this doc if you meet new questions about `How to` and find the answers!
+
+## Backbone network using MMClassification
+
+The Hierarchy Registry function proposed in MMCV allows modules in various algorithm libraries of OpenMMLab to call each other. Users can use backbone networks from MMClassification in MMDetection without implementing a network that already exists in MMClassification.
```

Comment:
The model registry in MMDet, MMCls, MMSeg all inherits from the root registry in MMCV. This allows these repositories to directly use the modules already implemented by each other. Therefore, users can use backbone networks from MMClassification in MMDetection without implementing a network that already exists in MMClassification.